vrf.go

```diff
@@ -5,6 +5,8 @@ import (
 	r255 "github.com/gtank/ristretto255"
 )
 
+var kusama bool = true
```

Comment:
Having a global state like this is a bad idea. Probably we can refactor it later and keep it inside some struct.

Reply:
yeah I don't really like it either but I basically copied this from rust schnorrkel. I'm gonna change it to be a key option actually